File: CityGuess/app/src/main/java/pl/edu/pwr/lab46/i236764/PreferencesHelper.java
package pl.edu.pwr.lab46.i236764;

import android.content.Context;
import android.content.SharedPreferences;

public class PreferencesHelper {

    private final String PREFS_NAME = "pl.edu.pwr.lab46.i236764.preferences";
    private Context context;

    public PreferencesHelper(Context context) {
        this.context = context;
    }

    public String getString(String key, String defaultValue) {
        SharedPreferences pref = context.getSharedPreferences(PREFS_NAME, 0);
        return pref.getString(key, defaultValue);
    }

    public void putString(String key, String value) {
        SharedPreferences pref = context.getSharedPreferences(PREFS_NAME, 0);
        SharedPreferences.Editor editor = pref.edit();
        editor.putString(key, value);
        editor.commit();
    }

}

File: CityGuess/app/src/main/java/pl/edu/pwr/cityguess/PreferencesHelper.kt
package pl.edu.pwr.cityguess

import android.content.Context

/**
 * Created by stevyhacker on 27.7.14..
 */
class PreferencesHelper(private val context: Context) {
    private val PREFS_NAME = "pl.edu.pwr.cityguess.preferences"
    fun getString(key: String?, defaultValue: String?): String? {
        val pref = context.getSharedPreferences(PREFS_NAME, 0)
        return pref.getString(key, defaultValue)
    }

    fun putString(key: String?, value: String?) {
        val pref = context.getSharedPreferences(PREFS_NAME, 0)
        val editor = pref.edit()
        editor.putString(key, value)
        editor.commit()
    }

}